webpack.config.js

```diff
@@ -0,0 +1,75 @@
+const path = require("path");
+const HtmlWebpackPlugin = require("html-webpack-plugin");
+const UglifyJsPlugin = require("uglifyjs-webpack-plugin");
+const MiniCssExtractPlugin = require("mini-css-extract-plugin");
+const OptimizeCSSAssetsPlugin = require("optimize-css-assets-webpack-plugin");
+
+module.exports = function (env, options) {
+  const isProduction = options.mode === "production";
```

Comment:
In scripts (in `package.json`) `env` is used and here `mode`.
`"build:dev": "webpack --env dev --progress --profile --colors"`

Please update webpack config to make use of `env`.